must-gather/collection-scripts/gather

```diff
@@ -24,6 +29,16 @@ fi
 # Call post-uninstall.sh
 post-uninstall.sh
 
-echo "deleting empty files" >> ${BASE_COLLECTION_PATH}/gather-debug.log
+# timestamp for ending of the script
+END_TIME="$(date +%r)"
+date2=$(date +"%s")
+diff=$("${date2}"-"${date1}")
+printf "collection ended at: %s \n" "${END_TIME}" >> ${BASE_COLLECTION_PATH}/gather-debug.log 2>&1
+
+# collecting the total time taken by the scrript script
+total_time=$(("${diff}" / 3600 )) hours $(("${diff}" % 3600) / 60)) minutes $(("${diff}" % 60)) seconds
```

Comment:
@crombus my CI is failing here, need some help in what pattern am I missing.

Reply:
Getting this alert
```
 ^-- SC1073: Couldn't parse this $((..)) expression. Fix to allow more checks.
 ^-- SC1009: The mentioned syntax error was in this $((..)) expression.
^-- SC1072: Expected a double )) to end the $((..)). Fix any mentioned problems and try again.                                                       
```

Reply:
This line is also not getting printed in the gather debug log file, probably due to this error that I'm getting?.

Reply:
total_time=$(("${diff}" / 3600 )) hours $((("${diff}" % 3600) / 60)) minutes $(("${diff}" % 60)) seconds

Reply:
update the brackets in minutes column.